src/WebpGenerator.php

```diff
@@ -23,6 +23,10 @@ public function generate(string $url, string $mimeType): string
             return $url;
         }
 
+        if (strpos($_SERVER['HTTP_ACCEPT'], 'image/webp') !== true ) {
```

Comment:
`strpos` never returns a boolean `true`, therefore this check will always be true. This should only happen when there is no `image/webp` in the accept header.

```suggestion
        if (strpos($_SERVER['HTTP_ACCEPT'], 'image/webp') === false) {
```